data/rentals.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "date": "2022-03-02",
+    "book": "Harry Potter",
+    "person": "Edi"
+  },
+  {
+    "date": "2021-04-02",
+    "book": "Lord of the Rings",
+    "person": "Ahmed"
+  },
+  {
+    "date": "124",
+    "book": "#<Book:0x000001add39fdb98>",
+    "person": null
+  },
+  {
+    "date": "12",
+    "book": "#<Book:0x000001add39fdb98>",
+    "person": "#<Teacher:0x000001add39fd378>"
+  }
```

Comment:
- [ ] Kindly fix the storing of rentals as now you store the object id of book and person and there is no way to retrieve the old rentals when the program is closed and run again.